azext_iot/operations/central.py

```diff
@@ -17,18 +16,26 @@ def find_between(s, start, end):
     return (s.split(start))[1].split(end)[0]
 
 
-def iot_central_device_show(
-    cmd, device_id, app_id, central_api_uri="api.azureiotcentral.com"
-):
-    sasToken = get_iot_hub_token_from_central_app_id(cmd, app_id, central_api_uri)
-    endpoint = find_between(sasToken, "SharedAccessSignature sr=", "&sig=")
-    target = {"entity": endpoint}
-    auth = BasicSasTokenAuthentication(sas_token=sasToken)
-    service_sdk, errors = _bind_sdk(target, SdkType.service_sdk, auth=auth)
-    try:
-        return service_sdk.get_twin(device_id)
-    except errors.CloudError as e:
-        raise CLIError(unpack_msrest_error(e))
+def iot_central_device_show(cmd, device_id, app_id, central_api_uri='azureiotcentral.com'):
+    from azext_iot.common._azure import get_iot_central_tokens
+
+    tokens = get_iot_central_tokens(cmd, app_id, central_api_uri)
+    exception = None
+
+    # try all hubs, return first exception if none succeed
```

Comment:
For comments, I usually think its better to include "why" rather than "how".

What is present in tokens?
What are we checking all hubs for?
Why are we checking all hubs?
Why are we returning the first exception rather than all of them?